src/SignalR/clients/ts/signalr/README.md

```diff
@@ -4,6 +4,7 @@ JavaScript and TypeScript clients for SignalR for ASP.NET Core
 
 ```bash
 npm install @aspnet/signalr
+# or "yarn add @aspnet/signalr"
```

Comment:
Whats with using `#` here?

Reply:
It's a code comment. Users should run either npm install or yarn add, but not both. Open to suggestions for making that more obvious

Reply:
```bash
npm install @aspnet/signalr
```
or
```bash
yarn add @aspnet/signalr
```

Could work

Reply:
ok, updated the readme